contrib/util/ccda_import/import_ccda.php

```diff
@@ -75,6 +76,7 @@ function showHelp(): void
     echo "\n";
     echo "Example:\n";
     echo "  php import_ccda.php --sourcePath=/path/to/import/documents \\\n";
+    echo "                      --authName=admin \\\n";
     echo "                      --site=default \\\n";
     echo "                      --openemrPath=/var/www/openemr \\\n";
     echo "                      --isDev=true \\\n";
```

Comment:
would add a flag that this is running via the web page (then can ensure it will work in web page mode without breaking the original command line mode)

Reply:
this is required when not in dev mode. without it documents can't be saved to patient documents which should be the normal import mode so the original ccda can be available to user as a reference.
Sine there isn't a required order for args on command line unclear how it breaks anything.
In the end Brady authName can't go away otherwise import breaks in normal mode.

Reply:
As long as ordering doesn't matter and authName is not required when isDev=true, then i am good :)

Reply:
I've tested both options and all works well. When isDev=true I'll still pass authUser as session with the default but regardless, the $_SESSION['authUser'] is never looked at.

I'm happy I learned this feature and will now recommend to all seeking moving to OpenEMR.
Thanks for creating!